code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -22,30 +22,29 @@
 	energy = 50
 
 
-/obj/effect/accelerated_particle/New(loc)
-	..()
-
-	addtimer(CALLBACK(src, .proc/move), 1)
-
-
-/obj/effect/accelerated_particle/Bump(atom/A)
-	if(A)
-		if(isliving(A))
-			toxmob(A)
-		else if(istype(A, /obj/machinery/the_singularitygen))
-			var/obj/machinery/the_singularitygen/S = A
-			S.energy += energy
-		else if(istype(A, /obj/singularity))
-			var/obj/singularity/S = A
-			S.energy += energy
-		else if(istype(A, /obj/structure/blob))
-			var/obj/structure/blob/B = A
-			B.take_damage(energy * 0.6)
-			movement_range = 0
-
-/obj/effect/accelerated_particle/Crossed(atom/A, oldloc)
+/obj/effect/accelerated_particle/Initialize(loc)
+	. = ..()
+	addtimer(CALLBACK(src, .proc/propagate), 1)
+	RegisterSignal(src, COMSIG_CROSSED_MOVABLE, .proc/try_irradiate)
+	RegisterSignal(src, COMSIG_MOVABLE_CROSSED, .proc/try_irradiate)
+	QDEL_IN(src, movement_range)
+
+/obj/effect/accelerated_particle/proc/try_irradiate(src, atom/A)
 	if(isliving(A))
-		toxmob(A)
+		var/mob/living/L = A
+		L.apply_effect((energy * 6), IRRADIATE, 0)
+	else if(istype(A, /obj/machinery/the_singularitygen))
+		var/obj/machinery/the_singularitygen/S = A
+		S.energy += energy
+	else if(istype(A, /obj/structure/blob))

```

Comment:
New strat just aim the pa at the blob core

Reply:
I was going to comment on that but it's just a lot of setup. It would work though and make a 3-wide space. but so much work to setup

Reply:
If you could get cargo on the ball, you could order a PA crate to move the entire thing to the blob, but yeah it would be so much work that it's probably not a viable option